G_channels.md

```diff
@@ -50,7 +50,7 @@ The transport layer is where the rubber meets the road. The `Phoenix.Channel.Tra
 
 - Transport Adapters
 
-The default transport mechanism is via WebSockets which will fall back to LongPolling if WebSockets are not available/working. Other transport adapters are possible, and indeed, we can write our own if we follow the adapter contract. Please see `Phoenix.Transports.WebSocket` for an example.
+The default transport mechanism is via WebSockets which will fall back to LongPolling if WebSockets are not available or not working. Other transport adapters are possible, and we can write our own if we follow the adapter contract. Please see `Phoenix.Transports.WebSocket` for an example.
```

Comment:
It only falls back if `window.WebSocket` is not defined, or if you explicitly tell it to use LongPoll, so the existing sentence is correct